code/backend/batch/utilities/helpers/processors/Process.py

```diff
@@ -0,0 +1,14 @@
+from ..AzureBlobStorageClient import AzureBlobStorageClient
+from .DocumentProcessorHelper import DocumentProcessor
+from .IntegratedVectorizationProcessorHelper import (
+    IntegratedVectorizationProcessorHelper,
+)
+
+
+class Process:
```

Comment:
This is a factory class so maybe rename this to EmbedderFactory and the method should probably be "create" as it doesn't get one, it creates one each time. "Process" for me is a low level concept and doesn't describe what its creating

Reply:
Updated